packages/indexer-agent/src/network.ts

```diff
@@ -135,8 +135,13 @@ export class Network {
     let pending = true
     let output: providers.TransactionReceipt | undefined = undefined
 
-    const estimatedGas = await gasEstimation()
-    const txPromise = transaction(Math.ceil(estimatedGas.toNumber() * 1.5))
+    const estimatedGasPrice = await gasEstimation()
+    const paddedGasPrice = Math.ceil(estimatedGasPrice.toNumber() * 1.5)
```

Comment:
Is the 1.5 always fixed?

Reply:
Yeah. 

I think it would be useful to parameterize this; I was thinking that I will make that update in a subsequent PR to avoid bike shedding on this one and because I don't want to keep adding complexity to the transaction management config that is already new to indexers.

Reply:
Could be interesting to explore controlling this parameter using "slow", "normal", "fast", "rapid" categories on startup for the entire agent, or by applying them per deployment rule.